pulsar-common/src/main/proto/PulsarApi.proto

```diff
@@ -477,6 +484,8 @@ message CommandAck {
 
     optional uint64 txnid_least_bits = 6 [default = 0];
     optional uint64 txnid_most_bits = 7 [default = 0];
+
+    repeated BatchMessageIndexesAckData batch_message_ack_indexes = 8;
```

Comment:
If we added an `ackSet` bitset to MessageIdData, we don't need to introduce additional structure, no?